concepts/conditionals/.meta/config.json

```
@@ -0,0 +1,5 @@
+{
+  "blurb": "With conditional functions you can check what's going on.",
```

Is this description accurate? Maybe slightly rephrase to something like:

```suggestion
  "blurb": "With conditional functions you can conditionally execute code.",
```

Or is conditionals something else in Red?

---

It's certainly more accurate, but perhaps less understandable for somebody new to programming, which was my intention. Although, Exercism is made primarily for people knowing how to program, but wanting to learn new language. For them, this description is not necessary at all, because every programmer knows what a "conditional" is, so probably it does not matter what we write here :-)

---

> It's certainly more accurate, but perhaps less understandable for somebody new to programming

I'd argue that "check what's going on" actually is _less_ understandable as it could basically mean anything 🤷 

> Although, Exercism is made primarily for people knowing how to program

As a matter of fact, Exercism is made just for people that know how to program. We specifically expect people to already know programming :)

> probably it does not matter what we write here

Yeah, this is true.

---

```suggestion
  "blurb": "With conditional functions you can execute different pathways based on the inputs.",
```

Maybe?

---

I was just about to accept Erik's idea, because he convinced me. Now I have to choose... :-)

---

> > every programmer knows what a "conditional" is, so probably it does not matter what we write here
> Yeah, this is true.

I was thinking about languages, that seem to not have conditionals – like SQL, HTML... maybe there are some more in Exercism's set of tracks? Perhaps we could imagine a programmer not knowing this concept. How should we explain it to such person?

---

Conditional branching is part of Turning-completeness, so I think any actual programming language has it (e.g. SQL does, but HTML which is a markup language, not a programming language, doesn't). I think as a blurb it's safe to go with either Erik or my suggestions :)

---

Ok, so erik wins for being first and shorter. Thanks!